libs/data-types/src/lib/decorators/NonEmptyString.ts

```diff
@@ -10,7 +10,9 @@ export function NonEmptyString(userOptions: Partial<TypeDecoratorOptions> = {}):
     return (target: Object, propertyKey: string | symbol) => {
         const options = mixinDefaultTypeDecoratorOptions(userOptions);
 
-        WithValidation(IsStringWithNonzeroLength(), options)(target, propertyKey);
+        WithValidation(IsString({ each: options.isArray }), options)(target, propertyKey);
```

Comment:
This is a bit awkward. It seems there are multiple decorators. Maybe we should wrap the multiple decorators into one in the validation lib.

Reply:
done